content/en/dashboards/ddsql_editor/reference/aggr_functions.md

```diff
@@ -0,0 +1,99 @@
+---
+title: AGGR Functions in DDSQL
+---
+
+{{< callout url="https://google.com">}}
+DDSQL is in private beta.
+{{< /callout >}}
+
+<div class="alert alert-warning">AGGR functions are experimental, and subject to change.</div>
+
+`AGGR` functions are functions that can be used in the `AGGR` statement and executed by the metrics query system. `AGGR` functions take one parameter, which can be either a string name of the metric being queried, or a [DQLExpr][1] subquery (for [multilayer aggregation][2]).
```

Comment:
this seem more technical detail (metrics query system) - something like AGGR functions are used for querying metrics would be simpler here.

Reply:
This is just the reference for functions that can be used with `AGGR`, not the `AGGR` statement itself, which is why we're describing the potential parameters and their use cases here. Users will mostly come here from the higher-level `AGGR` statement explanation, so they should already be aware of what `AGGR` does. 

I'll add a link to the `AGGR` statement in this first sentence, though, because I realized it's not clear from the content of this page that its content is nested under the broader `AGGR` explanation.